ActyvityLog.js

```diff
@@ -127,15 +352,32 @@ render() {
         style = {styles.progres}
         progressTintColor = '#674DCD'
         progress = {0.5}/>
-      <ScrollView style = {styles.scrollView}>
-        <Text></Text>
-      </ScrollView>
+      {this._renderListView()}
     </View>
   );
 }
 }
 
 const styles = StyleSheet.create({
+  padding : {
```

Comment:
Please move all the styles to separate file like ActyvityLogSyles.js and include in this one